css/card.css

```diff
@@ -66,28 +283,104 @@
     "head  head"
     "card  text"
     "card  text";
+  overflow-y: auto;
 }
 
 [role="dialog"][aria-hidden="true"] {
   display: none;
 }
 
+[role="dialog"][aria-hidden="noshow"] {
+  opacity: 0;
+}
+
+@media screen and (max-width: 1400px) {
+  #gacha-controls tcg-card {
+    --card-width: 300px;
+    --card-height: 419px;
+  }
+
+  .details-dialog-card {
+    width: 25vw;
+    height: calc(25vw*1.395);
+  }
+}
+
 @media screen and (max-width: 1000px) {
+
+  #gacha-controls tcg-card {
+    --card-width: 200px;
+    --card-height: 279px;
+  }
+
   #card-details-dialog {
     grid-template-areas:
       ".     x"
       "head  head"
       "text  text"
       "card  card";
-    overflow: auto;
+    overflow-y: auto;
+    overflow-x: hidden;
+    justify-items: center;
   }
 
   .details-dialog-text {
     /* Override overflow: auto for small screens, and let whole dialog scroll instead of just the text section */
     overflow: visible;
+    min-height: auto;
+    padding-bottom: 20px;
+  }
+
+}
+
+@media screen and (max-width: 900px) {
+
+  section label {
+    white-space: nowrap;
+    margin-top: 10px;
+    display: block;
+    padding-left: 0px;
+  }
+  section#gacha-controls label {
+      white-space: normal;
+      margin-top: 0px;
+      display: inline-block;
+      padding-left:0px
+  }
+
+}
+
+@media screen and (max-width: 800px) {
+  #gacha-controls #card-list {
+    min-height:55vh;
+  }
+  #gacha-controls.pile-display tcg-card {
+    left: 0;
+    right: 0;
+    margin: 3% auto 0 auto;
+  }
+  #gacha-controls.pile-display #card-list tcg-card.clicked {
+    animation: bottom 1s ease 0s 1;
+    animation-fill-mode: forwards;
+  }
+  @keyframes bottom {
+    0% {
+      margin-top: 3%;
+    }
+    100% {
+      margin-top: calc(var(--card-height) + 3% + 100px);
+    }
+  }
+}
+
+@media screen and (max-width: 550px) {
+  #gacha-display-toggle {
+    float:none;
```

Comment:
Is float: none needed? I can't find anywhere that is setting float

Reply:
Sorry, again something I tried and then deleted and forgot to remove from here too.